docs/production-deployment/cloud/service-health.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: service-health
+title: Service health
+sidebar_label: Service health
+description: Use Temporal Cloud metrics to monitor your production deployment Service health.
+toc_max_heading_level: 4
+tags:
+  - temporal-cloud
+  - metrics
+  - observability
+  - health
+---
+
+Temporal Cloud metrics help you to monitor the Service health of your production deployments.
+
+## Monitor availability issues
+
+When you see a sudden drop in Worker resource utilization, verify whether Temporal Cloud's API is showing an increased error rate.
+
+### Reference Metrics
+
+- [temporal\_cloud\_v0\_service\_latency\_bucket](/production-deployment/cloud/metrics/reference#temporal_cloud_v0_service_latency_bucket)
```

Comment:
nitpik:

It may worth mentioning this metric only include a subset of the operations that we consider mission critical:
`SignalWithStartWorkflowExecution`, `SignalWorkflowExecution`, `StartWorkflowExecution`.

But nevertheless, they are representative for the possible service outage or performance degradation on our side.

reference: https://github.com/temporalio/infra-chronosphere/blob/main/terraform/rollup_rules.tf#L259